src/common/api/authService.ts

```diff
@@ -45,6 +50,16 @@ export const authApi = baseApi.injectEndpoints({
           },
         }),
     }),
+    getMe: builder.query<AuthResults['getMe'], AuthParams['getMe']>({
+      query: ({ token }) =>
```

Comment:
Pull token from app state by default? `store.getState()`

Reply:
This causes a circular dependency, so I'm going to leave it for now and we can return to this later.